build-system/test-configs/forbidden-terms.js

```diff
@@ -304,6 +304,10 @@ const forbiddenTermsGlobal = {
       'extensions/amp-a4a/0.1/amp-ad-template-helper.js',
       'extensions/amp-analytics/0.1/instrumentation.js',
       'extensions/amp-analytics/0.1/variables.js',
+      // These files reference a service defined within the same extension, so it's okay
+      // to use getServicePromiseForDoc() for it.
+      'extensions/amp-form/0.1/amp-form.js',
+      'extensions/amp-form/0.1/form-dirtiness.js',
```

Comment:
nit: As written, this comment looks like it applies to lines 309-322 when it likely only applies to lines 309-310. Here's a suggested alternative for better readability.
```suggestion
      'extensions/amp-form/0.1/amp-form.js', // References service defined in amp-form.
      'extensions/amp-form/0.1/form-dirtiness.js', // References service defined in amp-form.
```